coderadar-rest/src/main/java/io/reflectoring/coderadar/rest/analyzerconfig/CreateAnalyzerConfigurationController.java

```diff
@@ -24,7 +24,7 @@ public CreateAnalyzerConfigurationController(
   }
 
   @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity addAnalyzerConfiguration(
+  public ResponseEntity<Object> addAnalyzerConfiguration(
```

Comment:
Maybe return `ResponseEntity<HttpStatus>` instead of `ResponseEntity<Object>` and do error handling in the `ExceptionHandlerController` ?

Reply:
I'll work on the ErrorHandler in the next PR